src/routes/safe/components/Apps/utils.ts

```diff
@@ -151,6 +152,12 @@ export const staticAppsList: Array<StaticAppInfo> = [
   },
 ]
 
+export const getAppsList = async (): Promise<AppData[]> => {
+  const result = await fetchAppList()
```

Comment:
there's no need to catch a potential HTTP request error?

Reply:
I added the catch and we already have the fallback to the static list there :+1: